src/branch.coffee

```diff
@@ -11,17 +11,25 @@ export default (test, consequent, alternate = (props) -> props) ->
         consequent
         Component
       )
-      if not Consequent.displayName? or Consequent.displayName is 'ret'
-        Consequent.displayName = "branch(#{Component.displayName ? ''})"
+      Consequent.displayName = "branch(#{
+        if not Consequent.displayName? or Consequent.displayName is 'ret'
+          Component.displayName ? ''
```

Comment:
In what case would a `displayName` not be present? (Would it make sense to add the new display name here if it's missing?)

Reply:
I see, looks like this is being handled below.